src/relcast.erl

```diff
@@ -450,7 +450,7 @@ in_flight(ForActorID, State = #state{pending_acks = Pending}) ->
     P = maps:get(ForActorID, Pending, []),
     Active = lists:filter(fun ({_Ref, CF, _Key, _Multicast}) ->
                                   CF == State#state.active_cf
-                          end, P),
+                          end, lists:reverse(P)),
```

Comment:
I don't think that you need to do this, filter leaves the list in the same order it had before.